### PR TITLE
CEDS-1504 Remove legacy mapping

### PR DESCRIPTION
**Pull request by PauloPortugal**
* Remove legacy declaration mapping
* Run ScalaFmt

### CODE REVIEW

File: app/connectors/CustomsDeclareExportsConnector.scala [hunk 1]
```diff
@@ -115,5 +116,19 @@ class CustomsDeclareExportsConnector @Inject()(
...
       }
+}
 
+object CustomsDeclareExportsConnector {
+  def toXml(metaData: MetaData): String = {
+    import java.io.StringWriter
+
+    import javax.xml.bind.{JAXBContext, Marshaller}
+
+    val jaxbMarshaller = JAXBContext.newInstance(classOf[MetaData]).createMarshaller
```
Comment (bambuchaAdm) [inline app/connectors/CustomsDeclareExportsConnector.scala]: This is quite heavy operation - can we move this Val to object and not build marshaller every time ?

Comment (bambuchaAdm) [reply app/connectors/CustomsDeclareExportsConnector.scala]: Ok - from [Internet](https://timjansen.github.io/jarfiller/guide/jaxb/multithreading.xhtml) we must not share marshaller but Context is perfectly fine.

File: app/models/ExportsDeclaration.scala [hunk 2]
```diff
@@ -46,13 +46,17 @@ case class ExportsDeclaration(
...
-
   def updatedItem(itemId: String, update: ExportItem => ExportItem): ExportsDeclaration =
     itemBy(itemId).fold(this) { item =>
       val updated = update(item)
       copy(items = items.filterNot(_.id.equalsIgnoreCase(itemId)) + updated)
     }
+
+  def itemBy(itemId: String): Option[ExportItem] = items.find(_.id.equalsIgnoreCase(itemId))
+
+  def lrn(): Option[String] = this.consignmentReferences.map { _.lrn }
```
Comment (bambuchaAdm) [inline app/models/ExportsDeclaration.scala]: maybe just 'val' ?

Comment (PatrykRudnicki) [inline app/models/ExportsDeclaration.scala]: For one line map better is use normal parenthesis instead of curly

Comment (PauloPortugal) [reply app/models/ExportsDeclaration.scala]: addressed

Comment (PauloPortugal) [reply app/models/ExportsDeclaration.scala]: addressed

**Pull request merged**